src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -59,6 +59,11 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('sku', arg_type=get_enum_type(IotDpsSku),
                    help='Pricing tier for the IoT provisioning service.')
         c.argument('unit', help='Units in your IoT Provisioning Service.', type=int)
+        c.argument(
+            "tags",
+            options_list=["--tags"],
+            help="IoT dps instance tags. Property bag in key-value pairs with the following format: a=b;c=d."
```

Comment:
Could this structure of tags be consistent with the tags of other CLI commands?
For example:
```
az resource tag --ids {resource-id} --tags key1=value1 key2=value2 key3=value3
```
This tag verification can be referred to [code link](https://github.com/Azure/azure-cli/blob/master/src/azure-cli/azure/cli/command_modules/resource/_params.py#L101)

Reply:
Hmm I guess we aren't aligned on the tags input pattern since IoT hasn't had tags support in the CLI until recently and with the first IoT service for tags upon creation we went in [this direction](https://docs.microsoft.com/en-us/cli/azure/ext/azure-iot/dt?view=azure-cli-latest#ext_azure_iot_az_dt_create-examples).

Reply:
Ok, we will conform the --tags to be consistent.

Reply:
Fixed this